modular_skyrat/modules/emote_panel/code/emote_panel.dm

```diff
@@ -0,0 +1,978 @@
+/mob/proc/manipulate_emotes()
+	if(!mind)
+		return
+	var/list/available_emotes = list()
+	var/list/all_emotes = list()
+
+	// code\modules\mob\emote.dm
+	var/list/mob_emotes = list(
+		/mob/proc/emote_flip,
+		/mob/proc/emote_spin
+	)
+	all_emotes += mob_emotes
+	if(istype(src, /mob))
+		available_emotes += mob_emotes
+
+	// code\modules\mob\living\emote.dm
+	var/list/living_emotes = list(
+		/mob/living/proc/emote_blush,
+		/mob/living/proc/emote_bow,
+		/mob/living/proc/emote_burp,
+		/mob/living/proc/emote_choke,
+		/mob/living/proc/emote_cross,
+		/mob/living/proc/emote_chuckle,
+		/mob/living/proc/emote_collapse,
+		/mob/living/proc/emote_cough,
+		/mob/living/proc/emote_dance,
+		/mob/living/proc/emote_drool,
+		/mob/living/proc/emote_faint,
+		/mob/living/proc/emote_flap,
+		/mob/living/proc/emote_aflap,
+		/mob/living/proc/emote_frown,
+		/mob/living/proc/emote_gag,
+		/mob/living/proc/emote_giggle,
+		/mob/living/proc/emote_glare,
+		/mob/living/proc/emote_grin,
+		/mob/living/proc/emote_groan,
+		/mob/living/proc/emote_grimace,
+		/mob/living/proc/emote_jump,
+		/mob/living/proc/emote_kiss,
+		/mob/living/proc/emote_laugh,
+		/mob/living/proc/emote_look,
+		/mob/living/proc/emote_nod,
+		/mob/living/proc/emote_point,
+		/mob/living/proc/emote_pout,
+		/mob/living/proc/emote_scream,
+		/mob/living/proc/emote_scowl,
+		/mob/living/proc/emote_shake,
+		/mob/living/proc/emote_shiver,
+		/mob/living/proc/emote_sigh,
+		/mob/living/proc/emote_sit,
+		/mob/living/proc/emote_smile,
+		/mob/living/proc/emote_sneeze,
+		/mob/living/proc/emote_smug,
+		/mob/living/proc/emote_sniff,
+		/mob/living/proc/emote_stare,
+		/mob/living/proc/emote_strech,
+		/mob/living/proc/emote_sulk,
+		/mob/living/proc/emote_sway,
+		/mob/living/proc/emote_tilt,
+		/mob/living/proc/emote_tremble,
+		/mob/living/proc/emote_twitch,
+		/mob/living/proc/emote_twitch_s,
+		/mob/living/proc/emote_wave,
+		/mob/living/proc/emote_whimper,
+		/mob/living/proc/emote_wsmile,
+		/mob/living/proc/emote_yawn,
+		/mob/living/proc/emote_gurgle,
+		/mob/living/proc/emote_inhale,
+		/mob/living/proc/emote_exhale,
+		/mob/living/proc/emote_swear
+	)
+	all_emotes += living_emotes
+	if(istype(src, /mob/living))
```

Comment:
isliving()

Reply:
Appropriate check applied, also moved the checks to make it easier to read